stores/metadata.go

```diff
@@ -61,6 +51,17 @@ const (
 	contractStateFailed
 )
 
+var (
+	pruneSlabsAlertID = frand.Entropy256()
```

Comment:
`nit` this could live in a separate `alerts.go` with constructors for the alerts like we have in other places, e.g. `NewPruneSlabsAlert`